### DISCUSSION TITLE
I can't generate .sql in pscale command

### DISCUSSION THREAD

**Discussion by teerawit37** [planetscale/discussion#346] [Q&A]
Hi, I want to download backup file (.sql) all database in my account. Because, I want to handover database task to my team (I resigned). Can you tell me if can migrate database to other database or solution for generate .sql file (I can't generate sql file on mysql + pscale command )

**Comment by fooforge** [comment]
Hey @teerawit37,

You can use `pscale database dump <DATABASE> <BRANCH>` to dump your database. This will create a new folder in your working directory with separate files for the schema and your data.  See also [our documentation](https://planetscale.com/docs/reference/database) for the CLI reference.

Another option would be to simply use `mysqldump` which will dump your database into a single file. Personally, I like the CLI approach better, though.
